src/main/java/ch/njol/skript/effects/EffLeash.java

```diff
@@ -34,8 +34,11 @@
 import ch.njol.util.Kleenean;
 
 @Name("Leash entities")
-@Description("Leash living entities to other entities.")
-@Examples("leash the player to the target entity")
+@Description("Leash living entities to other entities. When trying to leash an Ender Dragon, Wither, Player, or a Bat, this effect will not work.")
```

Comment:
I actually think spigot javadocs reference is useful here as the list may change later on
```suggestion
@Description("Leash living entities to other entities. When trying to leash an Ender Dragon, Wither, Player, or a Bat, this effect will not work. See <a href="https://hub.spigotmc.org/javadocs/spigot/org/bukkit/entity/LivingEntity.html#setLeashHolder(org.bukkit.entity.Entity)">Spigot's Javadocs for more info</a>.")
```